CoreFoundation/PlugIn.subproj/CFBundle.c

```diff
@@ -714,7 +714,7 @@ static CFBundleRef _CFBundleCreate(CFAllocatorRef allocator, CFURLRef bundleURL,
 
     bundle->_url = newURL;
     
-#if !DEPLOYMENT_RUNTIME_OBJC && !DEPLOYMENT_TARGET_WINDOWS
+#if !DEPLOYMENT_RUNTIME_OBJC && !DEPLOYMENT_TARGET_WINDOWS && !defined(__ANDROID__)
```

Comment:
I'm thinking if, for consistency, there shouldn't be a `DEPLOYMENT_TARGET_ANDROID` that we could use. I'm also pretty sure there are other FHS bits — it should be entirely enabled or entirely disabled.

Reply:
What exactly is FHS?? There may be other areas I should be #ifdeffing out but this is the only one that was causing a crash.

Reply:
FHS is the Filesystem Hierarchy Spec, but in this context it refers to bundles that store contents pursuant that spec: https://forums.swift.org/t/proposal-foundation-allowing-for-system-bundles-on-linux/7417/6

Android is not a FHS system and should not have those on.